package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "start": "vite",
     "build": "tsc && vite build",
+    "deploy": "tsc && vite build && (cd cdk-out && cdk deploy)",
```

Comment:
You also can use --prefix flag to avoid using of cd command
```
"deploy" : "npm run build && npm --prefix ./cdk-out run deploy"
```

Reply:
ok thanks!